roles/undeploy_sro/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+sro_channel: "4.9"
```

Comment:
Does the channel need to be variable? The operator channel changes for OCP version

Reply:
This is a variable.  how about if I put update it to 4.10 for the 4.10 branch and so on.